test/nocks.ts

```diff
@@ -36,7 +36,7 @@ export function projectId(status: number|(() => string), reply?: () => string) {
     reply = status;
     status = 200;
   }
-  return nock('http://metadata.google.internal')
+  return nock('http://metadata.google.internal.')
```

Comment:
Can you use the exported HOST_ADDRESS instead?
https://github.com/googleapis/gcp-metadata/blob/0499ecde33a54a0adaeb4d3cbf964610687b0011/src/index.ts#L12

Reply:
Done